examples/js/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id("org.jetbrains.kotlin.js") version "1.4.0"
```

Comment:
`id("org.jetbrains.kotlin.js") version "1.4.0"` should be a part of com.kotlin.aws.js.plugin

Reply:
ok